<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Saha Housings - Start</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>280</width>
     <height>110</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="createHousingButton">
      <property name="text">
       <string>Create Housing</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signinHousingButton">
      <property name="text">
       <string>Sign in to a housing</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>